specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.0/examples/recoverStorageSasDefinition.json

```diff
@@ -0,0 +1,27 @@
+{
+   "parameters":{
+      "vaultBaseUrl":"https://storage-sdk-test.vault-int.azure-int.net/",
+      "storage-account-name":"recoversas01",
+      "sas-definition-name":"recoverStrgSasDef01",
+      "api-version":"7.0-preview"
+   },
+   "responses":{
+      "200":{
+         "headers":{
+
+         },
+         "body":{
+            "id":"https://storage-sdk-test.vault-int.azure-int.net/storage/recoversas01/sas/recoverStrgSasDef01",
+            "sid":"https://storage-sdk-test.vault-int.azure-int.net/secrets/recoversas01-recoverStrgSasDef01",
+            "templateUri":"se=2018-02-01T00%3A00Z&spr=https&sv=2017-04-17&sr=b&sig=XXFNfuMCHYrBx0bhemJ7PWn0xGfImMXT6LfbXWvtRUk%3D",
+            "sasType":"service",
+            "validityPeriod":"P15D",
+            "attributes":{
```

Comment:
The recoveryLevel attribute seems to be missing; is the service returning it?
[...]
I just checked the service code, and it seems that the controller is indeed setting the attribute. The recoveryLevel should be present with every response that returns the entity's attributes.

Reply:
Updated all the examples which return StorageAccountAttributes or SasDefinitionAttributes to include recoverLevel in the response.